src/pool/srv_target.c

```diff
@@ -2676,8 +2676,7 @@ ds_pool_tgt_discard_ult(void *data)
 		D_GOTO(free, rc = 0);
 	}
 
-	ex_status = PO_COMP_ST_UP | PO_COMP_ST_UPIN | PO_COMP_ST_DRAIN |
-		    PO_COMP_ST_DOWN | PO_COMP_ST_NEW;
+	ex_status = PO_COMP_ST_UP | PO_COMP_ST_UPIN | PO_COMP_ST_DRAIN | PO_COMP_ST_NEW;
```

Comment:
Oh, I meant to ask about this one, just so I can understand better (no change recommended).

So before, pool_child_discard() would be run for targets in DOWNOUT state only? But now, it will run for targets that are DOWN or DOWNOUT state (because PO_COMP_ST_DOWN bit is not set in ex_status with this code change)?